File: DemoApp-Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/home/SectionHeader.java
package com.applovin.enterprise.apps.demoapp.data.home;

/**
 * A {@link ListItem} representing a section header on the home screen
 * <p>
 * Created by santoshbagadi on 2019-09-10.
 */
public class SectionHeader
        implements ListItem
{
    private final String title;

    public SectionHeader(final String title)
    {
        this.title = title;
    }

    /**
     * @return The time of the section header.
     */
    public String getTitle()
    {
        return title;
    }

    @Override
    public int getType()
    {
        return TYPE_SECTION_HEADER;
    }
}

File: DemoApp-Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/data/home/SectionHeader.kt
package com.applovin.enterprise.apps.demoapp.data.home

/**
 * A [ListItem] representing a section header on the home screen
 * <p>
 * Created by Harry Arakkal on 9/17/2019.
 */
data class SectionHeader(val title: String) : ListItem
{
    override val type: Int = ListItem.SECTION_HEADER
}
